### PR TITLE
[rllib] Update Docs for RLLib

### PR DESCRIPTION
**Pull request by richardliaw**
## What do these changes do?

Recent changes to `Tune` have resulted in changes in API for RLLib. This PR updates the documentation.

### CODE REVIEW

File: doc/source/rllib.rst [hunk 1]
```diff
@@ -15,22 +15,24 @@ It currently supports the following algorithms:
...
-You can run training with
+You can train an example DQN agent with the following command
 
 ::
 
-    python ray/python/ray/rllib/train.py --env CartPole-v0 --run PPO --config '{"timesteps_per_batch": 10000}'
+    python ray/python/ray/rllib/train.py --run DQN --env CartPole-v0
 
 By default, the results will be logged to a subdirectory of ``/tmp/ray``.
 This subdirectory will contain a file ``config.json`` which contains the
```
Comment (ericl) [inline doc/source/rllib.rst]: params.json now

File: doc/source/rllib.rst [hunk 2]
```diff
@@ -167,4 +195,15 @@ First create a cluster as described in `managing a cluster with parallel ssh`_.
...
+Using RLLib with Ray.tune
+-------------------------
+
+All Agents implemented in RLLib support the `Trainable` interface.
+
+Here is an example of using Ray.tune with RLLib:
+
+::
+
+    python ray/python/ray/rllib/train.py -f tuned_examples/cartpole-grid-search-example.yaml
```
Comment (ericl) [inline doc/source/rllib.rst]: Also show an equivalent python API example with run_experiments?

Comment (richardliaw) [reply doc/source/rllib.rst]: done.

**Review by ericl — COMMENTED** [review]
Could you also update the README to point to this? Might also want to remove the Athena docs, I'm not sure it works

**Review by ericl — APPROVED** [review]

**Pull request merged**